{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,7 +16,7 @@ known_django = django
 default_section = THIRDPARTY
```

Comment:
On line 6 of this file : Remove `{% if cookiecutter.virtualization_tool == 'drifter' %}virtualization/, {% endif %}`